okta/config.go

```diff
@@ -90,8 +92,17 @@ func (c *Config) loadAndValidate(ctx context.Context) error {
 		httpClient.Transport = transport.NewGovernedTransport(httpClient.Transport, apiMutex, c.logger)
 	}
 
+	var orgUrl string
+	var disableHTTPS bool
```

Comment:
I like where this is going with having a boolean to mark if going to be enabling TLS, or not.